src/Form/DeveloperSyncForm.php

```diff
@@ -46,8 +46,18 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#title' => $this->t('Sync developers'),
       '#open' => TRUE,
     ];
+    
+    $form['sync']['description'] = [
+      '#type' => 'html_tag',
+      '#tag' => 'p',
+      '#value' => $this
```

Comment:
Little nitpicking, but single method call should be in the same line, parameter list could be in multiple lines if it is necessary.